backend/app/management/commands/load_loksabha.py

```diff
@@ -0,0 +1,62 @@
+"""
+Django management command load_loksabha
+
+Updates local db with values from base csv dataset
+"""
+
+import os
+import pandas
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from app.models import LSElection
+
+# pylint: disable=R0801
+
+
+class Command(BaseCommand):
+    """
+    Custom django-admin command to load Lok Sabha data over the years
+    """
+
+    help = ""
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "dataset_name",
+            type=str,
+            action="store",
+            help="Name of dataset in app/data folder (with extension)",
+        )
+
+    def handle(self, *args, **options):
+        # pylint: disable=too-many-locals
+        file_name = options.get("dataset_name")
+        file_path = os.path.join(settings.DATASET_DIR, file_name)
+        df = pandas.read_csv(file_path)
+    # pylint: disable=R0801
+
+        for year, state_name, constituency_name, constituency_no, party_name, candidate, \
+                candidate_position, margin_percentage, vote_share in zip(
```

Comment:
I suggest using parenthesis around lines that are too long and distributing the contents in the parenthesis over multiple lines rather than using \ (even though the formatter might use \) since it's more general and flexible (easily change the number of lines the statement spans).